packages/cli/src/cli.ts

```diff
@@ -480,14 +487,15 @@ async function nonInteractive(features: Feature[], {
     const unpackerOutput = _unpackerOutput ?? (singleFeature ? outputBase : path.join(outputBase, defaultUnpackerOutputFolder))
     const unminifyOutput = _unminifyOutput ?? (singleFeature ? outputBase : path.join(outputBase, defaultUnminifyOutputFolder))
 
-    const perfOutputBase = singleFeature
+    const perfOutputBase = _perfOutput || (singleFeature
         ? features.includes(Feature.Unpacker) ? unpackerOutput : unminifyOutput
-        : findCommonBaseDir([unpackerOutput, unminifyOutput]) ?? outputBase
+        : findCommonBaseDir([unpackerOutput, unminifyOutput]) ?? outputBase)
     const perfOutputPath = path.join(perfOutputBase, 'perf.json')
 
     const outputPathsToCheck = []
     if (features.includes(Feature.Unpacker)) outputPathsToCheck.push(unpackerOutput)
     if (features.includes(Feature.Unminify)) outputPathsToCheck.push(unminifyOutput)
+    if (features.includes(Feature.Perf)) outputPathsToCheck.push(perfOutputPath)
```

Comment:
So this one should be 
```suggestion
    outputPathsToCheck.push(perfOutputPath)
```